### PR TITLE
Update feature-multifactor-unlock.md

### PR DESCRIPTION
**Pull request by HEVARY**
Added the client OS version in the missing section.

## Why

- Closes #[Issue Number]

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/feature-multifactor-unlock.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The policy setting has three components:
 ## Configure unlock factors
 
 > [!CAUTION]
-> On Windows 11, when the [DontDisplayLastUserName](/windows/security/threat-protection/security-policy-settings/interactive-logon-do-not-display-last-user-name) security policy is enabled, it is known to interfere with the ability to use multi factor unlock.
+> On Windows 10 and Windows 11, when the [DontDisplayLastUserName](/windows/security/threat-protection/security-policy-settings/interactive-logon-do-not-display-last-user-name) security policy is enabled, it is known to interfere with the ability to use multi factor unlock.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/feature-multifactor-unlock.md:33]:
```suggestion
> On Windows 10 and Windows 11, when the [DontDisplayLastUserName](/windows/security/threat-protection/security-policy-settings/interactive-logon-do-not-display-last-user-name) security policy is enabled, it is known to interfere with the ability to use multifactor unlock.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @HEVARY, kindly also commit the minor additional change. Many thanks.

File: windows/security/identity-protection/hello-for-business/feature-multifactor-unlock.md (hunk 1 above)
Comment (paolomatarazzo) [inline windows/security/identity-protection/hello-for-business/feature-multifactor-unlock.md:33]: The article applies to Win10/11 so we can remove the OS version here
```suggestion
> When the [DontDisplayLastUserName](/windows/security/threat-protection/security-policy-settings/interactive-logon-do-not-display-last-user-name) security policy is enabled, it is known to interfere with the ability to use multi factor unlock.
```

**Review by paolomatarazzo — CHANGES_REQUESTED** [review]
Thank you for the feedback, please review my comment with the changes.

**Pull request merged**